packages/elements/src/store.ts

```diff
@@ -261,31 +261,35 @@ export class PhotonClientStore {
   }
 
   private async checkSession() {
-    await this.sdk.authentication.checkSession();
-    const authenticated = await this.sdk.authentication.isAuthenticated();
-    this.setStore('authentication', {
-      ...this.store.authentication,
-      isAuthenticated: authenticated,
-    });
-    const user = await this.sdk.authentication.getUser();
-    const hasOrgs = !!this.sdk?.organization && !!user?.org_id;
-
-    let permissions: Permission[];
     try {
-      const token = await this.sdk.authentication.getAccessToken();
-      const decoded: { permissions: Permission[] } = jwtDecode(token);
-      permissions = decoded?.permissions || [];
+      await this.sdk.authentication.checkSession();
+      const authenticated = await this.sdk.authentication.isAuthenticated();
+      this.setStore('authentication', {
+        ...this.store.authentication,
+        isAuthenticated: authenticated
+      });
+      const user = await this.sdk.authentication.getUser();
+      const hasOrgs = !!this.sdk?.organization && !!user?.org_id;
+
+      let permissions: Permission[];
+      try {
+        const token = await this.sdk.authentication.getAccessToken();
+        const decoded: { permissions: Permission[] } = jwtDecode(token);
+        permissions = decoded?.permissions || [];
+      } catch (err) {
+        permissions = [];
+      }
+
+      this.setStore('authentication', {
+        ...this.store.authentication,
+        user: user,
+        isLoading: false,
+        isInOrg: authenticated && hasOrgs && this.sdk.organization === user.org_id,
+        permissions: permissions || []
+      });
     } catch (err) {
-      permissions = [];
+      console.log('Error checking photon session: ', err);
```

Comment:
`console.error`?

Reply:
ya good call, this is where I had a sentry error capture... replaced this with:
```
      this.setStore('authentication', {
        ...this.store.authentication,
        isLoading: false
      });
```